docs/validator/staking.md

```diff
@@ -132,3 +132,18 @@ If you would like to see how much a validator is staking, you can run the comman
   storage_usage: 182
 }
 ```
+
+## Automatically re-staking
+
+This is implemented as a separate service that runs in parallel with `nearcore` and re-stakes if it sees that given validator might be kicked out or is already kicked out.
+
+Note that right now this uses `validator_key.json` to sign transactions as well.  We plan to add an option for a different signing key soon.
+
+```bash
+cargo build -p restaked --release
+./target/release/restaked \
+    --home=/path/to/validator_key.json \
```

Comment:
let's use `--home=<path to folder with validator key>` because it's path to folder not to file.

Reply:
✅